edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/EdgeOperator.cs

```diff
@@ -708,18 +719,21 @@ V1PodTemplateSpec GetPodFromModule(Dictionary<string, string> labels, Kubernetes
             {
                 //pod labels
                 var podLabels = new Dictionary<string, string>(labels);
+
+                // pod annotations
+                var podAnnotations = new Dictionary<string, string>();
+                podAnnotations.Add(Constants.k8sEdgeOriginalModuleId, module.ModuleIdentity.ModuleId);
+                // Convert docker labels to annotations because docker labels don't have the same restrictions as
+                // Kuberenetes labels.
                 if (moduleWithDockerConfig.Config.CreateOptions?.Labels != null)
                 {
                     foreach (KeyValuePair<string, string> label in moduleWithDockerConfig.Config.CreateOptions?.Labels)
                     {
-                        podLabels.Add(label.Key, label.Value);
+                        Console.WriteLine($"Annotation: {KubeUtils.SanitizeAnnotationKey(label.Key)}, {label.Value}");
```

Comment:
Did you mean to leave this `Console.WriteLine` here?

Reply:
Nope!  Thanks!